.gitignore

```diff
@@ -19,6 +19,7 @@
 *~
 .DS_Store
 .vagrant
+.venv/
```

Comment:
Suggestion: maybe refer to https://github.com/github/gitignore/blob/7e6cd454e651aa28c810cb9a89eda9efb29593f7/Python.gitignore#L130